src/de/jost_net/JVerein/gui/menu/BuchungMenu.java

```diff
@@ -71,9 +72,12 @@ public BuchungMenu(BuchungsControl control)
     }
     addItem(new CheckedContextMenuItem("Buchungsart zuordnen",
         new BuchungBuchungsartZuordnungAction(control), "view-refresh.png"));
-    if (geldkonto)
+    if (geldkonto) {
       addItem(new CheckedContextMenuItem("Sollbuchung zuordnen",
-        new BuchungSollbuchungZuordnungAction(control), "view-refresh.png"));
+              new BuchungSollbuchungZuordnungAction(control), "view-refresh.png"));
```

Comment:
Diese Stelle hier im Menü finde ich etwas unlogisch. Das ist ja mitten in den Zuordnungen. 

Ich würde vorschlagen entsprechend der Idee aus #408 das Löschen nach Auflösen zu verschieben. Dann einen Seperator. Dann sollte das Mitglied anzeigen direkt danach kommen oder als letztes.

Reply:
Das Item könnte auch bei Anlagenbuchungen angezeigt werden also ohne if (geldkonto)

Reply:
1. Die Stelle wurde so gewählt, da es dann direkt unter Sollbuchung zuordnen steht und das für mich logisch zusammengehört
2. Warum kann ich dann nur Sollbuchungen bei Geldkonten zuordnen?

Reply:
Mit 2. hast du Recht, bei Anlagenbuchungen gibt es keine Sollbuchungen. Dann kann man auch keine Sollbuchung finden und die baraucht man ja.

Zu 1. Für einen Benutzer ist das aber nicht unbedingt klar. Woher soll er wissen, dass wir intern eine Sollbuchung brauchen um das Mitglied zu finden. Es könnte ja auch direkt gehen. Aus Sicht der Benutzer würde ich also nicht auf so einen Zusammenhang bauen. Ich klicke Buchung und will Mitglied. Es schaut halt auch nicht schön aus wenn es da zwischen den Icons der der Zuordnungen ist.